.github/workflows/accuracy_checker.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     strategy:
       matrix:
         runs-on: [ubuntu-20.04, ubuntu-22.04]
-        python-version: [3.7, 3.8, 3.9, '3.10']
+        python-version: [3.8, 3.9, '3.10']
```

Comment:
do you support 3.11?

Reply:
We do. There's just pinned pytest in tests which is too old